modules/instrument_manager/test/TestPlan.md

```diff
@@ -1,12 +1,10 @@
-Instrument Manager Test Plan
+# Instrument Manager Test Plan
 
 1. User has access to Instrument Manager if he/she has permission “superuser”.
    [Automation Testing]
-2. Check that tab exists on a default install (for superusers).
+2. Check that `Instrument Manager` menu-item exists on a default install under Admin menu-item (for superusers).
    [Automation Testing]
-3. Check that the warning text above is accurate for both with and without the directory writable by Apache. The select file/upload button should only be there if it's writable.
-   [Manual Testing]
-4. Verify GUI is updated, menu tabs are working.
+3. Check that the warning text is accurate if the `project/instruments` or `project/sql_tables` directories are not writable by Apache. The select file/upload panel should only be there if it's writable.
```

Comment:
Point 5 below is misnumbered now.. and I don't know if it makes sense for point `2` to be part of the test plan. The LORIS menu isn't really part of the module.

Reply:
Though I guess (other than the misnumbering) that's also outside of the scope of this PR.

Reply:
I agree for the menu except for the fact that it is the usual entry point. Do we have a LORIS Menu test suit?

Reply:
No, but we probably should.